mdapi/MetricsDiscoveryHelper.cpp

```diff
@@ -334,7 +334,8 @@ void MDHelper::SetMetricSetFiltering( TMetricApiType apiMask )
 /************************************************************************/
 void MDHelper::GetMetricsFromReport(
     const char* pReportData,
-    std::vector<TTypedValue_1_0>& results )
+    std::vector<TTypedValue_1_0>& results,
+	std::vector<TTypedValue_1_0>* maxValues)
```

Comment:
Passing a pointer to a vector feels a bit icky.  Is there any way to avoid this?  One option is to store the max values data in the same result vector, similar to the way it is currently storing both the "metrics" and "information".

Reply:
The function provided by MD works like this, that it puts metrics + information into the first vector in the argument and max values into the second vector (if not null).

We can have two local vectors in the function and then merge them into the final one. Or if we can offset somehow the `data()` in the vector.